### PR TITLE
Azure Reserved Instances Policy

### PR DESCRIPTION
**Pull request by ryanoleary**
### Description

This adds a new policy for reporting on underutilized Azure Reserved Instances

### Issues Resolved

none

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [x ] New functionality has been documented in the README if applicable
- [x ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Comment by ryanoleary** [comment]
Note that testing was performed against an Azure EA that had RIs.

File: cost/azure/reserved_instances/utilization/azure_reserved_instance_utilization.pt [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+script "azure_ri_request", type: "javascript" do
+  parameters "enrollment"
+  result "request"
+  code <<-EOS
+
+    var date = new Date();
+    date.setDate(date.getDate() - 3);
+    var enddate = date.toISOString().slice(0,10);
+    date.setDate(date.getDate() - 7);
+    var startdate = date.toISOString().slice(0,10);
```
Comment (douglaswth) [inline cost/azure/reserved_instances/utilization/azure_reserved_instance_utilization.pt]: There should probably be a comment explaining this date math.

**Comment by ryanoleary** [comment]
@cdwilhelm  i can't reply in your comment thread, but do you mean the Azure API access? If so, I don't think we need to comment on that because that key always has this access as far as I know. That key provides a broad range of access to the EA APIs.

**Comment by cdwilhelm** [comment]
@ryanoleary ok, sounds fine.  thanks.

**Pull request merged**